testdata/testoutput15

```diff
@@ -111,10 +111,10 @@ Minimum depth limit = 10
  3: ee
 
 /(*LIMIT_MATCH=12bc)abc/
-Failed: error 160 at offset 17: (*VERB) not recognized or malformed
+Failed: error 160 at offset 16: (*VERB) not recognized or malformed
```

Comment:
This is more logical now: the first bad character in the LIMIT_MATCH construct is at offset 16, not offset 17.